minst/model.py

```diff
@@ -1,15 +1,24 @@
-
+import copy
 import json
 import jsonschema
+import logging
 import pandas as pd
 import os
+from sklearn.cross_validation import train_test_split
 
 import minst.utils as utils
 
+logger = logging.getLogger(__name__)
+
+
+class MissingDataException(Exception):
+    pass
+
 
 class Observation(object):
     """Document model each item in the collection."""
```

Comment:
blargh github collapsed it out and i didn't notice.

Maybe the class docstring could just have a high-level flyby semantic description of what the point of the class is for? 

I think that I am confused about exactly what an observation is, or is intended to be in this context. The sort of annotations we make with our annotation tool are onsets, which are not exactly quite this sort of Observation (start_time yes... duration is I guess the next start_time minus the current one?), so the Observation here is an second-level abstraction.

But... maybe an observation is just the metadata on the note-files, in which case I'm not sure that start_time and duration make sense anymore.